lib/Onyx.js

```diff
@@ -28,6 +29,9 @@ const evictionBlocklist = {};
 // Optional user-provided key value states set when Onyx initializes or clears
 let defaultKeyStates = {};
 
+// Connections can be made before `Onyx.init`. They would wait for this promise before resolving
+const deferredInit = createDeferredTask();
```

Comment:
Can we rename this to `deferredInitTask` if we are creating a task when we call the method? The comment mentions that it's a promise. But after reviewing the code I see it's more than just a promise.